components/06-examples/beheer-basis.config.yml

```diff
@@ -0,0 +1,60 @@
+notes: |
+status: wip
+context:
+  menu: 
+    modifier: main
+    items: 
+      - label: Menu titel A
+        active: true
+      - label: Menu titel B
+      - label: Menu titel C
+  breadcrumbs:
+    crumbs:
+      - label: Home
+        url: '#'
+      - label: Menu titel A
+        url: '#'
+  title: Overzicht samenwerkingen
+  sections:
+  - title: Woning
+    id: panel1
+  - title: Bouwwerkzaamheden
+    id: panel2
+  - title: Plaats van het bouwwerk
+    id: panel3
+    open: true
+    formLegend: Aanvraagformulier
+    fieldsets:
+      - legend: Plaats van het bouwwerk
+        groups:
+          - id: omgevingsvergunningPostcode
+            inputType: input
+            type: text
+            label: Postcode
+            size: 8
+          - id: omgevingsvergunningHuisnummer
+            inputType: input
+            type: text
+            label: Huisnummer
+            size: 8
+          - id: omgevingsvergunningToevoeging
+            inputType: input
+            type: text
+            label: Huisnummer toevoeging
+            size: 8
+          - id: omgevingsvergunningStraatnaam
+            inputType: input
+            type: text
+            label: Straatnaam
+          - id: omgevingsvergunningWoonplaats
+            inputType: input
+            type: text
+            label: Woonplaats
+    submitButton:
+      type: submit
+      modifier: primary
+      label: Volgende stap
+  - title: Afmetingen van het bouwwerk
```

Comment:
Idem, ook een copy/paste overblijfsel?

Reply:
Idem. is inderdaad overblijfsel copy/paste. Ga ik opschonen.